src/main/java/foundationgames/enhancedblockentities/EnhancedBlockEntities.java

```diff
@@ -72,4 +73,14 @@ public static void load() {
             EBESetup.setupRRPShulkerBoxes();
         }
     }
+
+    public static void addPacks() {
+        RRPCallback.AFTER_VANILLA.register(packs -> {
+            packs.add(ResourceUtil.getPack());
+
+            ExperimentalSetup.cacheResources(MinecraftClient.getInstance().getResourceManager());
```

Comment:
Experimental packs should not be added this way, they need to be injected after all resource packs including user applied ones (hence the mixin to reloadable resource manager). This is so that they can properly generate particle textures from applied packs.